src/rest.js

```diff
@@ -27,24 +27,18 @@ function checkLiveQuery(className, config) {
 // Returns a promise for an object with optional keys 'results' and 'count'.
 function find(config, auth, className, restWhere, restOptions, clientSDK) {
   enforceRoleSecurity('find', className, auth);
-  return triggers.maybeRunQueryTrigger(triggers.Types.beforeFind, className, restWhere, restOptions, config, auth).then((result) => {
-    restWhere = result.restWhere || restWhere;
-    restOptions = result.restOptions || restOptions;
```

Comment:
@flovilmart here we have the same modification of where/options.

Reply:
yeah but it's before the RestQuery actually, not after. (ie: not mutating the internal state)